### ISSUE TITLE
No album art...

### ISSUE THREAD

**Issue by DARO75** [huynhsontung/Screenbox#97]
Audio files do not show album art!

**Comment by huynhsontung** [comment]
Screenbox only supports embedded image art at the moment. It doesn't support image art in the separate file in the directory or online album art lookup just yet. Although I do plan to implement them in the future.

**Comment by DARO75** [comment]
**Screenbox only supports embedded image art at the moment**
you test it? it does not work!

**Comment by huynhsontung** [comment]
Can you provide some context for this issue? What platform are you using? What are the file types of your audio files? Does the album art show up when you use Windows Explorer?

**Comment by Tonda102** [comment]
The embedded album images indeed do work, but only in the **Recent** under the **Home** tab:
![obrazek](https://github.com/huynhsontung/Screenbox/assets/80787721/9a499055-040a-4c74-b39f-fa0f879733d9)

Everywhere else it's missing:
- **Mini player** in the main screen
![obrazek](https://github.com/huynhsontung/Screenbox/assets/80787721/53843b39-2388-4b84-954c-850b544ef490)
- **Music player** screen
![obrazek](https://github.com/huynhsontung/Screenbox/assets/80787721/7a03a225-828a-43fd-8f39-9e9ab8fcc0c7)

Implementing it into the Windows's **volume control** widget would be nice too:
![Bez názvu](https://github.com/huynhsontung/Screenbox/assets/80787721/d158c9aa-f09c-48e5-a973-f9b3a6c8e685)

**Comment by huynhsontung** [comment]
@Tonda102 Interesting, the audio file is incorrectly recognized as a video file. That's why there is no album art shows up in the player. Can you provide the file extension for the file and the file metadata (Explorer File Properties -> Details tab)?

**Comment by Tonda102** [comment]
You're right, when I tried to play another audio file in **flac** format, the album image appeared:
![obrazek](https://github.com/huynhsontung/Screenbox/assets/80787721/9f64169f-7345-4500-a608-1ad801838779)
However it seems to be the only format which works.
Tried various mp3s, m4as, oggs and wavs and none of them worked.
Tried to convert this mp3 to other formats. This is how the result looks in the **Recent** tab:
![obrazek](https://github.com/huynhsontung/Screenbox/assets/80787721/7aa6d817-8c0c-40bb-8e13-4d59d313b7a2)
